themes/gbe-theme/components/_sayt.scss

```
@@ -0,0 +1,63 @@
+gbe-sayt {
+  background: var(--background-color-white);
+  border-bottom: var(--border-primary);
+  border-bottom-left-radius: var(--gbe-products-sayt-border-radius);
+  border-bottom-right-radius: var(--gbe-products-sayt-border-radius);
+  border-left: var(--border-primary);
+  border-right: var(--border-primary);
+  border-top: var(--border-primary);
+  box-sizing: border-box;
+  font-family: var(--font-family-text);
+  height: auto;
+  margin-top: -1px;
+  max-width: var(--gbe-search-box-input-and-sayt-max-width);
+  overflow: hidden;
+  padding: 0;
+  position: absolute;
+  width: var(--gbe-search-box-input-width, 400px);
+}
+
+.gbe-sayt-container {
+  padding: 0;
+}
+
+button.gbe-close {
+  position: absolute;
+}
+
+@media (max-width: 321px) {
+  .gbe-sayt-container gbe-products-sayt {
+    display: none;
+  }
+}
+// [closeText] {
+//   content: ''!important;
+//   color: red;
+// }
+
+[showclosebutton=""] {
+  .gbe-close {
+    font-size: 0;
+    right: 9px;
+    height: 40px;
+    width: 40px;
+    top: 9px;
+    padding: 0;
+    margin: 0;
+    -webkit-appearance: none;
+    border: none;
+    background-image: var(--gbe-sayt-clear-button-icon);
+    background-repeat: no-repeat;
+    background-position-x: 11px;
+    background-position-y: 6px;
+  }
+
+  gbe-products-sayt {
+    margin-top: 51px;
+  }
+
+  gbe-sayt svg.close-button-icon path {
+    fill: red!important;
```

```suggestion
    fill: red !important;
```

I don't suppose we can avoid the `!important`? Though it matters less here because it's inside an SVG.

---

this was a test for example, it has been removed in the next commit I'm about to make